File: Coding/Python/sum.py
# Program that asks the user for a number n and then will sum all the numbers from 1 to that number

print "Hello user please enter the number to sum to: "

n = raw_input()
sum_nums = 0

for i in range (0, int(n)+1):
    sum_nums += i
    
print "The sum of all the numbers from 1 to " + n + " is " + str(sum_nums) + " ."

File: Coding/Perl/sum.pl
# Program that will ask a user for a number(n) and then sum the numbers from 1 to n

print "Hello user please enter the number n to sum from 1 to n: ";
chomp($n=<>);

$sum = 0;

for($i=1;$i<=$n;$i++){
    $sum += $i;
}

print "The sum of the numbers 1 to n is $sum."